packages/perseus/src/__tests__/validation.typetest.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This file contains TypeScript type "tests" which ensure that types needed
+ * for scoring and validation stay in sync with other types in the system.
+ *
+ * If you make a change and one of these "satisfies" assertions fails, that
+ * will usually mean you've made a change that will cause runtime breakages in
+ * scoring or validation.
+ */
+import type {PerseusRenderer} from "../perseus-types";
+import type {RubricMap, ValidationDataMap} from "../validation.types";
+
+// We can use a 'widgets' map from a PerseusRenderer as a ValidationDataMap
+0 as any as PerseusRenderer["widgets"] satisfies ValidationDataMap;
+
+// We can use a RubricMap as a ValidationDataMap
+0 as any as RubricMap satisfies ValidationDataMap;
```

Comment:
May want to update these to `ScoringMap` since we're removing the `rubric` terminology.